cookbooks/aws-parallelcluster-platform/test/controls/sudo_config_spec.rb

```diff
@@ -9,7 +9,7 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'sudo_configured' do
+control 'tag:config_sudo_configured' do
```

Comment:
same here: we can rename it to tag:config_sudo

Reply:
Sure.